RoddyCore/src/de/dkfz/roddy/execution/io/ExecutionService.groovy

```diff
@@ -62,16 +62,21 @@ abstract class ExecutionService implements BEExecutionService {
         if (runMode == RunMode.CLI) {
             boolean isConnected = executionService.tryInitialize()
             if (!isConnected) {
-                int queryCount = 0
+                int passwordQueryCounts = 0
                 //If password is not stored, ask once for the password only, increase queryCount.
-                if (!Boolean.parseBoolean(Roddy.applicationConfiguration.getOrSetApplicationProperty(runMode, Constants.APP_PROPERTY_EXECUTION_SERVICE_STORE_PWD, Boolean.FALSE.toString()))) {
-                    Roddy.applicationConfiguration.setApplicationProperty(runMode, Constants.APP_PROPERTY_EXECUTION_SERVICE_AUTH_METHOD, Constants.APP_PROPERTY_EXECUTION_SERVICE_AUTH_METHOD_PWD)
+                boolean storePassword = Boolean.parseBoolean(Roddy.applicationConfiguration.
```

Comment:
Extracted complex expression into variable, to keep the `if` logic easier to understand.